test/appium/tests/atomic/account_management/test_wallet_management.py

```diff
@@ -309,3 +310,38 @@ def test_add_account_to_multiaccount_instance(self):
             self.driver.fail('Account was not added')
         if not account_button.color_matches('multi_account_color.png'):
             self.driver.fail('Account color does not match expected')
+
+    @marks.testrail_id(5406)
+    @marks.critical
+    def test_ens_username_recipient(self):
+        sign_in = SignInView(self.driver)
+        sign_in.create_user()
+
+        sign_in.just_fyi('switching to mainnet')
+        profile = sign_in.profile_button.click()
+        profile.switch_network('Mainnet with upstream RPC')
+        wallet = profile.wallet_button.click()
+
+        wallet.just_fyi('checking that "stateofus.eth" name will be resolved as recipient')
+        wallet.set_up_wallet()
+        wallet.accounts_status_account.click()
+        wallet.send_transaction_button.click()
```

Comment:
send_transaction_button already returns SendTransactionView no need to import SendTransactionView in this module for this, just `send_transaction wallet.send_transaction_button.click()` does the thing